### DISCUSSION TITLE
Nuova versione 1.5.7

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#627] [Annunci]
La nuova versione 1.5.7 comprende:
- correzione problemi scrutinio del primo periodo, anche per le classi articolate
- gestione studente con frequenza all’estero (tolto dalla visualizzazione in registro)
- aggiunte informazioni su studenti trasferiti (pagina assenze e pagine situazione studenti)
- gestione presa visione pagella (visibile da coordinatore in situazione-scrutini)
- migliorata visione voti da parte genitori (nome docente, divisione per tipo di prova)
- correzione errori vari

NB: la correzione delle procedure degli scrutini degli altri periodi sarà effettuata successivamente, in tempo utile per lo svolgimento degli scrutini.

**Comment by gianlucaaiello56** [comment]
salve @trinko , io inizio a svolgere gli scrutini da domani: posso installare la versione o ci sono problemi ?

**Comment by trinko** [comment]
Si, devi installare l'ultima versione.

**Comment by gianlucaaiello56** [comment]
ok @trinko ,quindi installando la versione potremo comunque svolgere gli scrutini. Giusto?

**Comment by trinko** [comment]
Si
